src/io/camera.js

```diff
@@ -99,31 +123,46 @@ module.exports = function Camera(options) {
     return console.error("An error occurred: [CODE " + error.code + "]");
   }
 
+  function falseSuccess(stream) {
+        //Stream for video processing
+        stream.getVideoTracks()[0].stop();
+  }
+
   // not doing anything now... for copying to a 2nd canvas
   function getSnapshot() {
-    var video;
-    // If the current context is WebRTC/getUserMedia (something
-    // passed back from the shim to avoid doing further feature
-    // detection), we handle getting video/images for our canvas 
-    // from our HTML5 <video> element.
-    if (webRtcOptions.context === "webrtc") {
+    if(options.version == 1){
```

Comment:
This all looks great! Please do try to standardize spacing -- there are some auto-formatters you can use too to achieve this! So, `if (options.version == 1) {` <= note the spaces. But not a big deal -- it just is nice over the long term (over years) to have the code be very regularly formatted. 

Thanks @Forchapeatl!!!

Reply:
Ohh, please do you mean indenting ?

Reply:
Ah no, actually just notice there's a space between `if` and `(` in line 109, but on 133 that's missing. it's a very small thing and not super important but is kind of nice to preserve completely consistent formatting as much as possible. Thank you!